### PR TITLE
Dev footer

### PR DESCRIPTION
**Pull request by sannesofie**
Added simple footer

### CODE REVIEW

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import './styles.css';
+import React from "react";
 
-export default function Footer () {
+const Footer = () => (
+<div className="footer">
```
Comment (AlinaTaoRao) [inline src/components/footer/Footer.js:5]: missing return(). please put jsx part into a return().
const Footer = () => {
   return()
}

**Review by AlinaTaoRao — CHANGES_REQUESTED** [review]

File: src/components/footer/Footer.js (hunk 1 above)
Comment (knnethdw) [reply src/components/footer/Footer.js:5]: @AlinaTaoRao This is the shorthand form, which is allowed. But you can, as a team, decide if you want every react component in the same style.

Comment (knnethdw) [reply src/components/footer/Footer.js:5]: Also, if you want a tool that can automate this: https://prettier.io/

Comment (knnethdw) [reply src/components/footer/Footer.js:5]: > @AlinaTaoRao This is the shorthand form, which is allowed. But you can, as a team, decide if you want every react component in the same style.

Btw, the shorthand form is only possible with the arrow function.

Comment (knnethdw) [reply src/components/footer/Footer.js:5]: > Which is the best practice?

There is not really a best practice. Just something you have to decide as a team.

**Review by AlinaTaoRao — APPROVED** [review]

**Pull request merged**